pkg/apis/core/validation/validation_test.go

```diff
@@ -3812,6 +3812,7 @@ func TestValidateVolumeMounts(t *testing.T) {
 		"name not found":                         {{Name: "", MountPath: "/foo"}},
 		"empty mountpath":                        {{Name: "abc", MountPath: ""}},
 		"relative mountpath":                     {{Name: "abc", MountPath: "bar"}},
+		"contain ':' in linux absolute path":     {{Name: "abc", MountPath: "/foo:bar"}},
```

Comment:
`/foo:bar` is perfectly valid directory name under Linux. What's purpose of this PR?

Reply:
Oh, I didn't notice that. Thanks for pointing out. I want to add a check so that user cannot add mount options in MountPath. Since ':' can be a valid character in linux file name, it can`t be done in API. I will close this pr.